middlewares/emailValidator.js

```diff
@@ -0,0 +1,25 @@
+const formatErrorMessage = require('../lib/utils/messageFormatter'),
+  { BAD_REQUEST_CODE } = require('../constants/index'),
+  { failureResponse } = require('../lib/utils/messageHandler'),
+  { validateEmail } = require('../lib/utils/validate');
+
+/**
+ * Rating validator middleware.
+ * @param {object} req - The request sent by user.
+ * @param {object} res - The response sent to user.
+ * @param {function} next - Next middleware to be called.
+ * @returns {object} The status of response and body of the response.
+ */
+const emailValidator = (req, res, next) => {
```

Comment:
I have a feeling that a function to validate email was created in the past and it was created by you. Why are you creating this function now?

Reply:
Yes I created the email function that can be called by any middleware